src/pages/sign-in/SignIn.tsx

```diff
@@ -31,11 +31,13 @@ import { Link } from "react-router-dom";
 import { PATH } from "../../constants/path.const";
 import useAccountStore from "../../zustand/useAccountStore";
 import { LOGO } from "../../images/images.const";
+import { showSignInMobileModal, SignInMobileModal } from "./SignInPhoneNumber";
 
 const SignIn = () => {
   const login = useAccountStore((state) => state.login);
   const clearErrors = useAccountStore((state) => state.clearErrors);
   const googleSignin = useAccountStore((state) => state.googleSignin);
+
   const loading = useAccountStore((state) => state.loading);
   const resError = useAccountStore((state) => state.errors);
```

Comment:
[37 - 42]: why don't you use destructuring ?

Reply:
Rerender purpose